lib/MusicBrainz/Server/Data/Release.pm

```diff
@@ -1004,31 +1004,34 @@ sub merge
     );
 
     if ($merge_strategy == $MERGE_APPEND) {
-        my %positions = %{ $opts{medium_positions} || {} }
+        my %positions_by_medium = %{ $opts{medium_positions} || {} }
```

Comment:
Non-functional change, just renaming the variable to be more clear what it's mapping, given that there's now a `%new_positions` variable.